### ISSUE TITLE
Add support for ARM64

### ISSUE THREAD

**Issue by mlhynfield** [rancherfederal/rke2-ansible#220]
Should be a pretty simple implementation, as far as my testing has gone you just need to parameterize wherever `amd64` is hardcoded.

**Comment by mddamato** [comment]
I agree there are some places where the file names should not be hard coded. It was only until recent that RKE2 supported ARM ( [v1.27.3](https://github.com/rancher/rke2/releases/tag/v1.27.3+rke2r1) ). What you're trying to do, however, can be supported by approaching this differently.

- For image tars use the  rke2_images_urls var to download from locations that contain the images you'd like to use
- For the rke2 binary use the rke2_tarball_url var to download from locations that have the rke2 binaries
- You can override any rpm repositories you'd like

I know this is not the cleanest workaround. The file names will still be marked as amd64 on the hosts and you can't let the playbook resolve versions, but this should work. If it doesn't let us know.

I'll leave this open for a future cleanup of hard coded file names and support for arm.

**Comment by aleiner** [comment]
addressed via #246
